lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -322,14 +316,27 @@ def set_aspect(self, aspect, adjustable=None, anchor=None, share=False):
         --------
         mpl_toolkits.mplot3d.axes3d.Axes3D.set_box_aspect
         """
-        if aspect != 'auto':
+        if aspect not in ['auto', 'equal']:
```

Comment:
```suggestion
        if aspect not in ('auto', 'equal'):
```
Minor thing, but slightly faster this way as this can be "compiled".